svm/Cargo.toml

```diff
@@ -71,6 +70,7 @@ rand0-7 = { workspace = true }
 shuttle = { workspace = true }
 solana-clock = { workspace = true }
 solana-compute-budget = { workspace = true, features = ["dev-context-only-utils"] }
+solana-compute-budget-instruction = { workspace = true }
```

Comment:
this would still make it difficult to separate the repo, right?

Reply:
If we can split the unit tests (which are acting more like integration tests now), we can remove this dev-dep as well.